src/main/java/pageObjects/loginPage.java

```diff
@@ -0,0 +1,92 @@
+package pageObjects;
+
+import driver.DriverFactory;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+
+public class loginPage extends DriverFactory {
```

Comment:
In Java, class names should start with an uppercase character https://www.tutorialspoint.com/how-do-i-write-class-names-in-java